internal/handlers/common/params.go

```diff
@@ -65,3 +67,24 @@ func AssertType[T types.Type](value any) (T, error) {
 
 	return res, nil
 }
+
+// AssertInt64Value asserts value's type returning protocol error for bad float values and unexpected results.
+// valueType parameter used to generate error message.
+func AssertInt64Value(valueType string, value any) (int64, error) {
```

Comment:
What you do here it's not an assertion, it's type casting if it suits. Shall the function be renamed, WDYT?

Reply:
`GetInt64Value`?